src/utils/numberInput.test.ts

```diff
@@ -0,0 +1,33 @@
+import { acceptNextDecimal, acceptNextInteger } from './numberInput'
```

Comment:
Should we move ./numberInput where it is used?
(NumberInput.tsx or similar)

Reply:
It was moved here temporarily when refactoring, I think it needs a new home

Reply:
Remove number input